esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/site/jasmin.cylc

```diff
@@ -25,12 +29,17 @@
         [[[directives]]]
             --mem = 2G
 
-    [[process<fast>]]
-        execution time limit = PT10M
+    # Resources for recipes that need more than the default.
+    # Variable (fast, medium) must be consistent with flow.cylc.
+    # Comment indicates example recorded usage on Jasmin.
+    [[process<fast=autoassess_landsurface_soilmoisture>]]
+        # Actual: 1m35s, 2.1 GB on 2024-03-29
+        execution time limit = PT3M
         [[[directives]]]
-            --mem = 2G
+            --mem = 3G
```

Comment:
Should this be `2G`? If so, should the directives from the specific recipe tasks be removed?

Reply:
It needs to be more than 2G since example recorded 2.1G.

Reply:
Apologies, I thought this was the `COMPUTE` section!